ios/brave-ios/Sources/AIChat/Components/Messages/AIChatMessageEditedLabelView.swift

```diff
@@ -0,0 +1,31 @@
+// Copyright 2024 The Brave Authors. All rights reserved.
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at https://mozilla.org/MPL/2.0/.
+
+import DesignSystem
+import SwiftUI
+
+struct AIChatMessageEditedLabelView: View {
+
+  let lastEdited: Date
+
+  private var lastEditedTime: String {
+    let formatter = DateFormatter()
+    formatter.dateFormat = "MMM d yyyy - h:mm a"
```

Comment:
Hmm does it have to be this format or is iso8601 fine?

Reply:
This was format from designs / desktop.